src/deepak/Test1_P1_NonRepeatingChar.java

```diff
@@ -0,0 +1,38 @@
+/*Program 1 : find the first non repeating character from given string. 
+   input : technocredits
+   output : h   
+*/
+
+package deepak;
+
+public class Test1_P1_NonRepeatingChar {
+
+	int getCharOccurence(String input, char ch) {
+		int count = 0;
+		for (int index = 0; index < input.length(); index++) {
+			char strChar = input.charAt(index);
+			if (strChar == ch) {
+				count++;
+			}
+		}
+		return count;
+	}
+
+	void getFirstNonRepeatChar(String input) {
+		char ch = ' ';
+		for (int index = 0; index < input.length(); index++) {
+			ch = input.charAt(index);
+			int count = getCharOccurence(input, ch);
+			if (count == 1) {
```

Comment:
here you can aslo use 
indexOf(ch) ==lastIndexOf(ch) logic to avoid one more for loop